spec/models/food_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe Recipe, type: :model do
+  describe 'associations' do
+    it 'belongs to a user' do
+      @user = User.new(email: 'a@a', password: '123456')
+      @food = Food.create(name: 'Doro', user_id: @user.id, unit: 'grams', quantity: 1)
+      expect(@food.user_id).to eq(@user.id)
+    end
+
+    it 'has many recipe foods' do
+      @user = User.create(email: 'a@a', password: '123456')
+      @recipe = Recipe.new(name: 'a', user: @user)
+      @food = Food.new(name: 'a', user_id: @user.id)
+      recipe_food = RecipeFood.create(quantity: 2, food: @food, recipe: @recipe)
+      expect(@recipe.recipe_foods).to include(recipe_food)
+    end
+
+    it 'returns all foods in alphabetical order' do
+      @user = User.create(email: 'a@a', password: '123456')
+      @food = Food.create(name: 'Doro', user_id: @user.id, unit: 'grams', quantity: 1)
+      @food2 = Food.create(name: 'Doro', user_id: @user.id, unit: 'grams', quantity: 1)
+      foods = Food.list_all_foods
+
+      expect(foods).to eq(foods.reverse)
```

Comment:
- [ ] Few of the test specs are failing. Kindly ensure all test cases are passing.

**Failing test cases:**
![image](https://user-images.githubusercontent.com/12745474/226097327-e1d549a9-3af2-4ce2-b64d-718d5a8843af.png)

Reply:
Solved.